CRM/Event/Form/Participant.php

```diff
@@ -1966,6 +1966,26 @@ public function getParticipantID(): ?int {
     return $this->_id ?: NULL;
   }
 
+  /**
+   * Get the contribution ID associated with the participant record.
+   *
+   *
+   * @api This function will not change in a minor release and is supported for
```

Comment:
Actually I have this test locally - I will put in a follow up as trying to get this to a mergeable state